homeassistant/components/sensor/__init__.py

```diff
@@ -88,6 +88,14 @@
 class SensorDeviceClass(StrEnum):
     """Device class for sensors."""
 
+    ACCUMULATED_PRECIPITATION = "accumulated_precipitation"
+    """Accumulared precipitation.
+
+    Unit of measurement: `UnitOfAccumulatedVolumtericFlux` enum
```

Comment:
Typo.
```suggestion
    Unit of measurement: `UnitOfAccumulatedVolumetricFlux` enum
```

... where is that enum, btw?

Reply:
The enum is in https://github.com/home-assistant/core/pull/81143

Reply:
IOW this shouldn't be merged before that 👍

Reply:
You are right - I have removed the reference so we don't need to wait for #81143